src/Components/ArticleVotes.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
+import {updateArticleVotes} from '../api';
+import  "./ArticleVotes.css"
+
+export default function ArticleVotes({article}) {
+  
+    const [votes, setVotes] = useState({inc_votes:0});
+    const [localVotes,setLocalVotes] =useState(0)
+    const [voted,setVoted] =useState(false)
+    const [error,setError] =useState(false)
+    
+    
+    const handleClick=  (event,num)=>{
+        event.preventDefault()
+        setVoted(true)
+        setVotes({inc_votes:num})
+        setLocalVotes((currVotes)=>{return currVotes +num})
+         
+        }
+        useEffect(() => {
+            updateArticleVotes(article.article_id, votes).catch(()=>{
+                setLocalVotes((currVotes)=>{return currVotes - votes.inc_votes})
+                setError(true)
+
+            });
+          }, [article.article_id, votes,]);
+  
+    
+        return (<div className="vote-section">
+        <h3>Please vote   </h3>
+        <p>Votes : {article.votes + localVotes}</p>
+        { !voted ? (<div> 
```

Comment:
nice job giving the user a message to say their vote was successful, I do think though you could allow them to change their vote and not disable all the buttons, eg a like cancels out a dislike